### ISSUE TITLE
Progress on the synthesis the purine analogue of OSM-S-106

### ISSUE THREAD

**Issue by mbhebhe** [OpenSourceMalaria/OSM_To_Do_List#558]
I am currently still trying to synthesise the purine analogue of OSM-S-106 to see if changing the scaffold improves the potency. 
The chlorination of hypoxanthine in the presence of [(MNB16)](https://au-mynotebook.labarchives.com/share/Tha%2520OSM%2520Series%25203/MTM3Ljh8MjcwNjkvMTA2L1RyZWVOb2RlLzM1NjkyMzM4Mzl8MzQ5Ljc5OTk5OTk5OTk5OTk1
), seems to work in the presence of N,N-dimethylaniline (DMA). The reaction does not go fully to completion, the product is water soluble and I am struggling to separate it from the starting material. I tried a small scale bromination on the pure-ish chloropurine. I tried with NBS at RT and this did not give the product. I tried with NBS and refluxing and that also didn't give the product. I tried using n-BuLi and bromine but that did not give the product either [(MNB18)](https://au-mynotebook.labarchives.com/share/Tha%2520OSM%2520Series%25203/MTQ2Ljl8MjcwNjkvMTEzL1RyZWVOb2RlLzg2NjUwMzM5MnwzNzIuOQ==). I will try again with a higher equivalence of n-BuLi.
If all works, I should be able to make the purine analogue of OSM-S-106 

![image](https://user-images.githubusercontent.com/26862286/34810979-865984aa-f6f2-11e7-8ffa-06ec5a7b413a.png)

**Comment by MFernflower** [comment]
random Q @mbhebhe Wouldn't: Methyllithium be more reactive thus meaning you need less to get your desired product?

**Comment by drc007** [comment]
Using one equiv of n-BuLi will simply remove the N-H, using 2 equiv may help but when trying to lithiate indoles use a 2-step process. Bergman, J.; Venemalm, L. (1992). "Efficient synthesis of 2-chloro-, 2-bromo-, and 2-iodoindole". J. Org. Chem. 57 (8): 2495.

**Comment by mbhebhe** [comment]
@MFernflower I thought methyllithium was less reactive than n-BuLi. I will try and use the chloropurine I have and if that fails I will buy some. Its so cheap too, don't know how I over looked that!
I think I will try @drc007 suggestion, I will use n-BuLi for the two step process instead of n-BuLi and tert-BuLi. I wonder if this will brominate at the 6th position on the pyrrole instead of the 7th position

**Comment by mcoster** [comment]
Yep, that's right, MeLi is less reactive than n-BuLi. Reactivity generally increases with substitution at the metallated position.

**Comment by MFernflower** [comment]
@mcoster Why is that? - I would have thought that the less stuff hanging off the metal would allow it to do it's job better? Unless the alkyl group needs to be repulsed by something for the reaction to work?? If so perhaps try Hexyllithium ?

**Comment by drc007** [comment]
@MFernflower Hexyllithium has a similar pKa to n-BuLi (approx 50), increased alkyl substitution e.g. t-BuLi gives increased basicity (approx 53).

**Comment by mbhebhe** [comment]
I purchased pure 6-chloropurine and currently trying to brominate with NBS.
If this doesn't work I will have use use titrated n-BuLi and bromine to brominate to make the [MNB18](https://au-mynotebook.labarchives.com/share/Tha%2520OSM%2520Series%25203/MTQ2Ljl8MjcwNjkvMTEzL1RyZWVOb2RlLzg2NjUwMzM5MnwzNzIuOQ==)
![image](https://user-images.githubusercontent.com/26862286/37806818-11474d92-2e96-11e8-922c-82374ec75e7c.png)
